astrodb_utils/fits.py

```diff
@@ -194,7 +197,9 @@ def check_header(header=None, format='simple-spectrum', ignore_simbad=False):
 
     coord = make_skycoord(header)
     if coord is None:
+        print("Coordinates are required")
         result = False
+        logger.debug(f"Coords: {coord}, result: {result}")
```

Comment:
Mixed use of print/logger. But more importantly, if coord is none then this will always log "Coords: None, result: False" so why have that? Did you intend for this not to be part of the if-statment?

Reply:
The mix of print and logger is intentional. This function is meant to be an "interactive" thing which folks use in Jupyter notebooks and what not. The print messages should display regardless of logger status.

The debug statement was added...for completeness. To make sure everything was going as expected.